scripts/framework-applications/suspend-suppliers-without-agreements.py

```diff
@@ -88,25 +102,30 @@
     supplier_ids = get_supplier_ids_from_args(args)
     suppliers = find_suppliers_without_agreements(client, framework_slug, supplier_ids)
 
-    with open(output_dir / FILENAME, 'w') as csvfile:
-        csv_headers = ['Supplier email', 'Supplier ID', "No. of services suspended"]
-
-        writer = csv.writer(csvfile, quoting=csv.QUOTE_MINIMAL)
-        writer.writerow(csv_headers)
+    framework_name = framework["name"]
+    contract_title = get_framework_contract_title(FRAMEWORKS_PATH, framework_slug)
 
-        for supplier in suppliers:
-            supplier_id = supplier["supplierId"]
-            framework_info = client.get_supplier_framework_info(supplier_id, framework_slug)
+    for supplier in suppliers:
+        supplier_id = supplier["supplierId"]
+        framework_info = client.get_supplier_framework_info(supplier_id, framework_slug)
 
+        logger.info(f"{prefix}Suspending services for supplier {supplier_id}")
+        if not dry_run:
             # Do the suspending
             suspended_service_count = suspend_supplier_services(
```

Comment:
It doesn't appear we are using this variable assignment? Can we remove it?

Reply:
Good point. It was logged before, I must have deleted a bit too much. I'll add that logging back in; I think it was useful